smolyay/basis.py

```diff
@@ -72,6 +84,32 @@ def in_domain(self, x):
             numpy.greater_equal(x, self.domain[0]), numpy.less_equal(x, self.domain[1])
         )
 
+    def _scale_to_domain(self, points, old_domain):
```

Comment:
I think this can be a public method:
```suggestion
    def scale_to_domain(self, points, old_domain):
```